src/opcodes/execution/log.rs

```diff
@@ -167,11 +167,16 @@ impl<const N: usize, E: VmEncodingMode<N>> DecodedOpcode<N, E> {
                     is_service: is_first_message,
                 };
 
-                let refund = vm_state.refund_for_partial_query(
-                    vm_state.local_state.monotonic_cycle_counter,
-                    &partial_query,
-                );
-                let ergs_refund = refund.refund();
+                let ergs_refund = if variant == LogOpcode::StorageRead {
```

Comment:
That was a whole point, that we always ask for some value, same as in circuit, and in circuit we require refund to be 0, but the number of accesses to oracle must match